python/tests/integration/arcticdb/version_store/test_basic_version_store.py

```diff
@@ -399,50 +367,50 @@ def test_prune_previous_versions_append_batch(lmdb_version_store):
     assert len(lib_tool.find_keys(KeyType.SYMBOL_LIST)) == 4
 
 
-def test_deleting_unknown_symbol(lmdb_version_store, symbol):
+def test_deleting_unknown_symbol(basic_store, symbol):
     df = sample_dataframe()
 
-    lmdb_version_store.write(symbol, df, metadata={"something": "something"})
+    basic_store.write(symbol, df, metadata={"something": "something"})
 
-    assert_frame_equal(lmdb_version_store.read(symbol).data, df)
+    assert_frame_equal(basic_store.read(symbol).data, df)
 
     # Should not raise.
-    lmdb_version_store.delete("does_not_exist")
+    basic_store.delete("does_not_exist")
 
 
-def test_negative_cases(lmdb_version_store, symbol):
+def test_negative_cases(basic_store, symbol):
     df = sample_dataframe()
     # To stay consistent with arctic this doesn't throw.
-    lmdb_version_store.delete("does_not_exist")
+    basic_store.delete("does_not_exist")
 
     # Creating a snapshot in an empty library should not create it.
-    lmdb_version_store.snapshot("empty_snapshot")
+    basic_store.snapshot("empty_snapshot")
     with pytest.raises(NoDataFoundException):
-        lmdb_version_store.delete_snapshot("empty_snapshot")
+        basic_store.delete_snapshot("empty_snapshot")
 
     with pytest.raises(NoDataFoundException):
-        lmdb_version_store.read("does_not_exist")
+        basic_store.read("does_not_exist")
     with pytest.raises(NoDataFoundException):
-        lmdb_version_store.read("does_not_exist", "empty_snapshots")
+        basic_store.read("does_not_exist", "empty_snapshots")
 
     with pytest.raises(NoDataFoundException):
-        lmdb_version_store.delete_snapshot("does_not_exist")
-    lmdb_version_store.write(symbol, df)
-    lmdb_version_store.delete(symbol)
-
-
-@pytest.mark.parametrize(
-    "lib_type",
-    [
-        "lmdb_version_store_v1",
-        "lmdb_version_store_v2",
-        "lmdb_version_store_no_symbol_list",
-        "s3_version_store_v1",
-        "s3_version_store_v2",
-    ],
-)
-def test_list_symbols_regex(request, lib_type):
-    lib = request.getfixturevalue(lib_type)
+        basic_store.delete_snapshot("does_not_exist")
+    basic_store.write(symbol, df)
+    basic_store.delete(symbol)
+
+# TODO: Do we really need to call this with these exact params?
```

Comment:
Def want to test this one with both symbol list on and symbol list off.

Reply:
The comment was more-so if we to test no_symbol_list fixture for s3, real-s3, etc....